tests/unit/alexnet_model.py

```diff
@@ -102,11 +102,24 @@ def cifar_trainset(fp16=False):
     if local_rank != 0:
         dist.barrier()
 
-    data_root = os.getenv("TEST_DATA_DIR", "/tmp/")
-    trainset = torchvision.datasets.CIFAR10(root=os.path.join(data_root, "cifar10-data"),
-                                            train=True,
-                                            download=True,
-                                            transform=transform)
+    if os.getenv("CIFAR10_OFFLINE", default=None):
```

Comment:
I am unfamiliar with these CIFAR10 env vars. Are they standard, and can you please provide doc links? Thanks!

Reply:
These are custom env vars. I have updated the logic in latest commit.

Reply:
We were facing issue to download CIFAR Dataset everytime so  we had stored it in local and were using the same for runs.